src/components/Footer.js

```diff
@@ -0,0 +1,138 @@
+                            xmlSpace="preserve"
+                        >
+                            <g>
+                            <path
+                                style={{ fill: '#fff' }}
+                                d="M37.2,64H28c-2.5,0-4.5-2-4.5-4.5V37H17c-2.5,0-4.5-2-4.5-4.5v-6.9c0-2.5,2-4.6,4.5-4.6h6.5v-5.5c0-9.1,6.2-15.5,15-15.5
+                                H47c1.4,0,2.5,1.1,2.5,2.5v13c0,1.4-1.1,2.5-2.5,2.5h-5.5v3h5.4c0,0,0.1,0,0.1,0c1.4,0.1,2.7,0.7,3.5,1.8c0.8,1.1,1.2,2.4,0.9,3.8
+                                    l-1.3,6.7C49.8,35.4,48,37,45.7,37h-4.3v22.5C41.5,62,39.5,64,37.2,64z M28.5,59h8V34.5c0-1.4,1.1-2.5,2.5-2.5h6.3l1.2-6H39
+                                    c-1.4,0-2.5-1.1-2.5-2.5v-8c0-1.4,1.1-2.5,2.5-2.5h5.5V5h-6c-6.1,0-10,4.1-10,10.5v8c0,1.4-1.1,2.5-2.5,2.5h-8.5v6H26
+                                    c1.4,0,2.5,1.1,2.5,2.5V59z"
+                            />
+                        </g>
+                        </svg>
+                        </a>
+                    </div>
+ 
+                    <div
+                        style={{
+                        height: '40px',
+                        width: '40px',
+                        display: 'inline-block',
+                        margin: '24px',
+                        }}
+                    >
+                        <a href="https://github.com/codeclannigeria" target="_blank">
+                        <svg 
+                        xmlns="http://www.w3.org/2000/svg" 
+                        viewBox="0 0 24 24"
+                        >
+                        <path 
+                            style={{ fill: "fff" }} 
+                            d="M12 2c5.514 0 10 4.486 10 10s-4.486 10-10 10-10-4.486-10-10 4.486-10 10-10zm0-2c-6.627 0-12 5.373-12 12s5.373 12 12 12 12-5.373 12-12-5.373-12-12-12zm0 6c-3.313 0-6 2.686-6 6 0 2.651 1.719 4.9 4.104 5.693.3.056.396-.13.396-.289v-1.117c-1.669.363-2.017-.707-2.017-.707-.272-.693-.666-.878-.666-.878-.544-.373.041-.365.041-.365.603.042.92.619.92.619.535.917 1.403.652 1.746.499.054-.388.209-.652.381-.802-1.333-.152-2.733-.667-2.733-2.965 0-.655.234-1.19.618-1.61-.062-.153-.268-.764.058-1.59 0 0 .504-.161 1.65.615.479-.133.992-.199 1.502-.202.51.002 1.023.069 1.503.202 1.146-.776 1.648-.615 1.648-.615.327.826.121 1.437.06 1.588.385.42.617.955.617 1.61 0 2.305-1.404 2.812-2.74 2.96.216.186.412.551.412 1.111v1.646c0 .16.096.347.4.288 2.383-.793 4.1-3.041 4.1-5.691 0-3.314-2.687-6-6-6z" 
+                        />
+                        </svg>
+                        </a>
+                    </div>
+
+                    <div
+                        style={{
+                        height: '40px',
+                        width: '40px',
+                        display: 'inline-block',
+                        margin: '24px',
+                        }}
+                    >
+                        <a href="https://twitter.com/codeclannigeria" target="_blank">
```

Comment:
Codacy found an issue: [Using target="_blank" without rel="noreferrer" is a security risk: see https://html.spec.whatwg.org/multipage/links.html#link-type-noopener](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=6058781)